### ISSUE TITLE
yapf inserst a blank line between function docstring and nested function declaration

### ISSUE THREAD

**Issue by Kamforka** [google/yapf#645]
Hi there,

I have an issue with `yapf` as it inserts a blank line between a function docstring and a nested function declaration, which is not compliant with [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).

**Code to reproduce:**

```python
def normal_function():
    """Return the nested function."""
    def nested_function():
        """Do nothing just nest within."""
        pass

    return nested_function
```

The code snippet above should be fine, however `yapf` reformats it as:
```python
def normal_function():
    """Return the nested function."""
                                         # <-- Added this blank line
    def nested_function():
        """Do nothing just nest within."""
        pass

    return nested_function
```

The blank line raises a linting error in my case.

I tried to use the knob `BLANK_LINE_BEFORE_NESTED_CLASS_OR_DEF` in my `setup.cfg` without any luck:

```
[yapf]
blank_line_before_nested_class_or_def=true
```

One workaround is that I switch off this linting rule, which I don't really wanna do if it can be fixed.

Can you please help me with this?

Version: 0.24.0

**Comment by Hiestaa** [comment]
Why was this closed with no comment? 

I ran into the same issue, I'm unable to get my flake8 and yapf config to agree on this. The linter does not want any blankline after the function docstring (`No blank lines allowed after function docstring (D202)`, but `yapf` cannot be configured to not add this blankline.

I sure can throw a `noqa` in there to disable the rule just for this case, but it would be nice to have a way to disable this somehow.

**Comment by gwelymernans** [comment]
The comment is "gwelymernans closed this in ba85e7c 24 days ago". So it's been fixed in top-of-tree.

**Comment by Hiestaa** [comment]
Oh thanks, my apologies. For some reason I skipped over the part that referenced the commit.
